src/densities.jl

```diff
@@ -112,11 +112,11 @@ end
 
 
 """
-Symmetrise a `RealFourierArray` by applying all symmetry operations of
-the bases (or all symmetries passed as the second argument) and forming
-the average. Notice that an identity needs to be passed explicitly.
+Symmetrize a `RealFourierArray` by applying all symmetry operations of
+the basis (or all symmetries passed as the second argument) and forming
+the average.
 """
-function symmetrize(ρin::RealFourierArray, ksymops=ksymops(ρin.basis))
+function symmetrize_density(ρin::RealFourierArray, ksymops=ksymops(ρin.basis))
```

Comment:
I called this symmetrize only on purpose because than we can also have a symmetrise function for wave functions at some point in the future.

Reply:
I changed it because symmetrize was a bit ambiguous to me, but now I think about it it's the only meaning of symmetrize I can think about. Will revert.